v3/lints/cabf_smime_br/lint_subscribers_crl_distribution_points_are_http.go

```diff
@@ -0,0 +1,63 @@
+/*
+ * ZLint Copyright 2023 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package cabf_smime_br
+
+import (
+	"net/url"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+func init() {
+	lint.RegisterLint(&lint.Lint{
+		Name:          "e_subscribers_crl_distribution_points_are_http",
+		Description:   "cRLDistributionPoints SHALL have URL scheme HTTP.",
+		Citation:      "7.1.2.3.b",
+		Source:        lint.CABFSMIMEBaselineRequirements,
+		EffectiveDate: util.CABF_SMIME_BRs_1_0_0_Date,
+		Lint:          NewSubscriberCrlDistributionPointsHTTP,
+	})
+}
+
+type subscriberCrlDistributionPointsHTTP struct{}
+
+func NewSubscriberCrlDistributionPointsHTTP() lint.LintInterface {
+	return &subscriberCrlDistributionPointsHTTP{}
+}
+
+func (l *subscriberCrlDistributionPointsHTTP) CheckApplies(c *x509.Certificate) bool {
+	return util.IsSubscriberCert(c) && (util.IsMultipurposeSMIMECertificate(c) || util.IsStrictSMIMECertificate(c))
```

Comment:
I think we need to handle Legacy certificates in this PR or acknowledge that the work has not yet been completed.

Reply:
IMO the challenge with that is that it leaves a gap in the requirement the lint is fulfilling. From purely a tracking perspective, we'd have to add another task to the checklist for handling the legacy version of the same lint.

Reply:
I updated this PR to handle legacy certificates.